plugins/tekton/README.md

```diff
@@ -41,28 +40,32 @@ This plugin will help with visualizing the workloads like Deployment, Replicaset
 
    ```
 
-5. To get the resources from a k8s cluster add the following annotations to the entity's catalog-info.yaml as K8s plugin identifies if the provided entity has k8s resources and if yes from which namespace it should get the resources based on the following annotations.
+5. To get the resources from a k8s cluster, add the following annotations to the entity's `catalog-info.yaml`:
+
    ```yaml
    annotations:
      backstage.io/kubernetes-id: <BACKSTAGE_ENTITY_NAME>
      backstage.io/kubernetes-namespace: <RESOURCE_NS>
    ```
-6. A custom label selector can also be added which will then be used by Backstage to find the resources. The label selector takes precedence over the id annotation.
+
+   The k8s plugin identifies if the provided entity contains k8s resources and from which namespace the plugin receives the resources based on the previous annotations.
+
+6. A custom label selector can also be added which is used by Backstage to find the resources. The label selector takes precedence over the ID annotation.
    `'backstage.io/kubernetes-label-selector': 'app=my-app,component=front-end`
-7. For k8s plugin to get the desired entity's k8s resources the following label should be added to the resources(if label selector is used then the labels mentioned in that should be present on the resource):
+7. For k8s plugin to get the desired entity's k8s resources, the following label must be added to the resources(if label selector is used then the labels mentioned in that must be present on the resource):
```

Comment:
```suggestion
7. Add the following label to the resources, so that the Kubernetes plugin gets the Kubernetes resources from the requested entity (when using the label selector, the labels mentioned must be present on the resource):
```

Reply:
Changed the sentence to:

_7. You must add the following label to the resources, so that the Kubernetes plugin gets the Kubernetes resources from the requested entity:
   ```yaml
   'backstage.io/kubernetes-id': <BACKSTAGE_ENTITY_NAME>`
   ```
    ---
    **NOTE**
    
    When using the label selector, the mentioned labels must be present on the resource.
    
    ---_